### DISCUSSION TITLE
It is displayed every time the page is reloaded, is the setting incorrect?

### DISCUSSION THREAD

**Discussion by ChinaGodzilla** [pyscript/pyscript#1183] [Q&A]
It is displayed every time the page is reloaded, is the setting incorrect?
![image](https://user-images.githubusercontent.com/43966525/218289841-de3e0d97-952a-45aa-b28a-b4b3c006e841.png)

**Comment by FabioRosado** [comment]
Are you trying to disable the splash screen? If so, `latest` doesn't contain that feature yet because we need to do a release.

That feature is in the unstable version which is pushed on each merge to main

**Comment by ChinaGodzilla** [comment]
thansk!Looking forward to the next version to remove this prompt.

**Comment by FabioRosado** [comment]
For now, you could try to hide the splashscreen with CSS, something like this might work but I haven't tested it yet:

```css
py-splashscreen {
display: none;
}
```

**Reply by ChinaGodzilla** [reply]
<style type="text/css">

    py-splashscreen {
    display: none;
    }
    </style>


@FabioRosado 

Thank you for successfully applying your method. I don't know when the kernel will load faster. Looking forward to pyscript getting better and better!
